src/main/java/com/github/nicholasmoser/gnt4/seq/comment/Functions.java

```diff
@@ -0,0 +1,319 @@
+      }
+      case Seqs.DOG_0000, Seqs.KAN_0000, Seqs.KIS_0000, Seqs.NEJ_0000, Seqs.TEN_0000 -> {
+        return 0x5D30;
+      }
+      case Seqs.ORO_0000, Seqs.SAR_0000, Seqs.ZAB_0000 -> {
+        return 0x5D40;
+      }
+      case Seqs.ANK_0000 -> {
+        return 0x5D50;
+      }
+      case Seqs.KID_0000, Seqs.SAS_0000 -> {
+        return 0x5D60;
+      }
+      case Seqs.KAK_0000, Seqs.TEM_0000 -> {
+        return 0x5D80;
+      }
+      case Seqs.KIM_0000, Seqs.SA2_0000 -> {
+        return 0x5DA0;
+      }
+      case Seqs.ITA_0000, Seqs.SKO_0000 -> {
+        return 0x5DB0;
+      }
+      case Seqs.TAY_0000 -> {
+        return 0x5E00;
+      }
+      case Seqs.BOU_0000 -> {
+        return 0x5E60;
+      }
+      case Seqs.NAR_0000 -> {
+        return 0x5EE0;
+      }
+      case Seqs.CHO_0000 -> {
+        return 0x5EF0;
+      }
+      case Seqs.HAK_0000 -> {
+        return 0x61A0;
+      }
+      default -> {
+        return 0x5CF0;
+      }
+    }
+  }
+
+  private static Map<Integer, Function> getChakraSubtractFunctions(String chrSeq) {
+    Map<Integer, Function> functions = new HashMap<>();
+    Function remove100Meter = new Function("Remove100%Meter", Collections.emptyList());
+    Function remove75Meter = new Function("Remove75%Meter", Collections.emptyList());
+    Function remove50Meter = new Function("Remove50%Meter", Collections.emptyList());
+    Function remove25Meter = new Function("Remove25%Meter", Collections.emptyList());
+    switch (chrSeq) {
+      case Seqs.KID_0000, Seqs.KIM_0000, Seqs.SA2_0000 -> {
+        functions.put(0x10E8, remove100Meter);
+        functions.put(0x1100, remove75Meter);
+        functions.put(0x1118, remove50Meter);
+        functions.put(0x1130, remove25Meter);
+      }
+      case Seqs.BOU_0000, Seqs.HI2_0000, Seqs.HIN_0000, Seqs.NAR_0000 -> {
+        functions.put(0x11D4, remove100Meter);
+        functions.put(0x11EC, remove75Meter);
+        functions.put(0x1204, remove50Meter);
+        functions.put(0x121C, remove25Meter);
+      }
+      case Seqs.CHO_0000 -> {
+        functions.put(0x1264, remove100Meter);
+        functions.put(0x127C, remove75Meter);
+        functions.put(0x1294, remove50Meter);
+        functions.put(0x12AC, remove25Meter);
+      }
+      case Seqs.DOG_0000, Seqs.KAR_0000 -> {
+        functions.put(0x10BC, remove100Meter);
+        functions.put(0x10D4, remove75Meter);
+        functions.put(0x10EC, remove50Meter);
+        functions.put(0x1104, remove25Meter);
+      }
+      case Seqs.SKO_0000 -> {
+        functions.put(0x114C, remove100Meter);
+        functions.put(0x1164, remove75Meter);
+        functions.put(0x117C, remove50Meter);
+        functions.put(0x1194, remove25Meter);
+      }
+      default -> {
+        functions.put(0x10DC, remove100Meter);
+        functions.put(0x10F4, remove75Meter);
+        functions.put(0x110C, remove50Meter);
+        functions.put(0x1124, remove25Meter);
+      }
+    }
+    return functions;
+  }
```

Comment:
If the functions look the same across the cast, wouldn't it be simpler (especially when implementing new functions) to pattern match them in the disassembling instead?

Reply:
Maybe, some of the functions have slight differences in a few of the characters. I think it would easier for some functions and a lot harder for others. I'm inclined to leave it as-is for now, but might look into changing it to be like that later. Feel free to make the change if you'd like to